core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -110,24 +106,30 @@ public InteractiveTransactionResponse withdraw(
     } catch (Exception ex) {
       throw new SepValidationException(String.format("invalid account: %s", sourceAccount), ex);
     }
-
+    Memo memo = makeMemo(withdrawRequest.get("memo"), withdrawRequest.get("memo_type"));
     String txnId = UUID.randomUUID().toString();
-    Sep24Transaction txn =
+    Sep24TransactionBuilder builder =
         new Sep24TransactionBuilder(txnStore)
             .transactionId(txnId)
-            .status(SepTransactionStatus.INCOMPLETE.toString())
-            .kind(Sep24Transaction.Kind.WITHDRAWAL.toString())
+            .status(INCOMPLETE.toString())
+            .kind(WITHDRAWAL.toString())
             .amountIn(strAmount)
             .amountOut(strAmount)
             .assetCode(assetCode)
             .assetIssuer(withdrawRequest.get("asset_issuer"))
             .startedAt(Instant.now().getEpochSecond())
             .stellarAccount(token.getAccount())
+            .stellarAccountMemo(token.getAccountMemo())
             .fromAccount(sourceAccount)
             .protocol(Sep24Transaction.Protocol.SEP24.toString())
-            .memoType(memoTypeString(MEMO_ID))
-            .domainClient(token.getClientDomain())
-            .build();
+            .domainClient(token.getClientDomain());
```

Comment:
domainClient should actually be clientDomain, right?

Reply:
Thanks a lot. Don't know why I wrote `domainClient`.

Fixed.